src/main/java/org/commcare/formplayer/services/MenuSessionRunnerService.java

```diff
@@ -307,8 +316,11 @@ public BaseResponseBean advanceSessionWithSelections(MenuSession menuSession,
                 }
             }
         } catch (CommCareSessionException ccse) {
-            notificationMessage = new NotificationMessage(ccse.getMessage(), true,
-                    NotificationMessage.Tag.menu);
+            // don't show a message in cases where the user is doing 'non-app' navigation
+            if (!nonAppNav) {
```

Comment:
Since we are now showing the screen prior to form entry we don't really expect an error to happen here. So curious why do we want to hide the errors for `nonAppNav` ?

Reply:
There can still be errors if the selected case was closed by the form or if the menu isn't visible anymore.

Reply:
makes sense 👍